app/controller/Users.py

```diff
@@ -12,43 +13,63 @@ def handle_get_user(self, user_id: int):
         user = self.users_service.get_user(user_id)
         return JSONResponse(
             status_code=status.HTTP_200_OK,
-            content=jsonable_encoder({
-                "message": user,
-                "status": status.HTTP_200_OK,
-            })
+            content=jsonable_encoder(
+                {
+                    "message": user,
+                    "status": status.HTTP_200_OK,
+                }
+            ),
         )
 
-    def handle_get_all_users(self):
-        users = self.users_service.get_all_users()
-        return JSONResponse(
-            status_code=status.HTTP_200_OK,
-            content=jsonable_encoder({
-                "message": users,
-                "status": status.HTTP_200_OK,
-            })
-        )
+    def handle_get_all_users(self, ids: list = None):
+        if ids:
+            users = self.users_service.get_users_by_ids(ids)
+            return JSONResponse(
+                status_code=status.HTTP_200_OK,
+                content=jsonable_encoder(
+                    {
+                        "message": users,
+                        "status": status.HTTP_200_OK,
+                    }
+                ),
+            )
+        else:
+            users = self.users_service.get_all_users()
+            return JSONResponse(
+                status_code=status.HTTP_200_OK,
+                content=jsonable_encoder(
+                    {
+                        "message": users,
+                        "status": status.HTTP_200_OK,
+                    }
+                ),
+            )

```

Comment:
este cambio, y en general los returns donde se separa el contenido de las llaves no me parecen muy necesarios. Creo que ya esta legible y agregarle mas lineas al archivo por esto no me parece muy worth it, no se que opinas

Reply:
dije muy lindo

Reply:
cambia2